### PR TITLE
create swap1108.md

### PR DESCRIPTION
**Pull request by swap1108**
Please review this

### CODE REVIEW

File: _contributors-app/swap1108.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+- |
+    - Activity recognition using smartphone sensors with auto-annotation in IoT: An innovative way for human acitvity recognition.
+
+- |
+    - Smart Home: ’Prediction of resident’s activities for smart living!’ A Concept derived from assisted living.
+
+- |
+    - Personal Financial Assistant: ’A Java desktop app which acts as a personal financial assistant!’· It provides an interface for users to save their financial data securely and manage the assets.
+
+About me: |
```
Comment (palash25) [inline _contributors-app/swap1108.md]: Do we have an **About me** field @knrt10 ?

**Comment by knrt10** [comment]
@swap1108 please fix the errors. And try to run locally and fix the errors then make a PR. steps to run tests are given in readme file.

**Comment by palash25** [comment]
Also @swap1108 after your are done with your changes commit them into the existing commit using
```
git add .
git commit --amend --no-edit

```

**Comment by swap1108** [comment]
Thanks @palash25  and @knrt10 . I will try to do so

File: _contributors-app/swap1108.md (hunk 1 above)
Comment (knrt10) [reply _contributors-app/swap1108.md]: If you want to add anything extra other than in template, you can add after the end of
```
---
```
line. cc @swap1108

**Comment by palash25** [comment]
I think it would be better if we open a new PR for this and close this one since @swap1108 hasn't amended the commits and has made 5 unnecessary commits this will fuck up our git log. What do you think @knrt10 ?

**Comment by swap1108** [comment]
Fine close it, i understand

**Comment by palash25** [comment]
@swap1108 I have closed it just make a new one with a fresh branch and copy paste the whole thing and please push with a single commit.

Thanks :+1:

**Pull request closed without merging**